examples/python/trigger-pagerduty-incident/README.MD

```diff
@@ -0,0 +1,96 @@
+### Get the example function
+
+Clone this repository which contains the example functions. 
+
+```bash
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/python/pagerduty
+git checkout master
+```
+
+### Customize the function
+
+For security reasons to not expose sensitive data we will create a Kubernetes [secret](https://kubernetes.io/docs/concepts/configuration/secret/) which will hold the routingkey information for the PagerDuty Event API v2. This secret will be mounted into the function during runtime. This is all taken care of for your by the appliance. We only have to create the secret with a simple command through `faas-cli`.
+
+First, change the configuration file [pdconfig.json](pdconfig.json) holding your sensitive pagerduty information in this folder:
+
+```json
+{
+    "routing_key": "<replace with your routing key>",
+    "event_action": "trigger" 
+}
+```
+
+Now go ahead and store this configuration file as secret in the appliance.
+
+```bash
+# set up faas-cli for first use
+export OPENFAAS_URL=https://VEBA_FQDN_OR_IP
+faas-cli login -p VEBA_OPENFAAS_PASSWORD --tls-no-verify # vCenter Event Broker Appliance is configured with authentication, pass in the password used during the vCenter Event Broker Appliance deployment process
+
+# now create the secret
+faas-cli secret create pdconfig --from-file=pdconfig.json --tls-no-verify
+```
+
+> **Note:** Delete the local `pdconfig.json` after you're done with this exercise to not expose this sensitive information.
+
+Lastly, define the vCenter event which will trigger this function. Such function-specific settings are performed in the `stack.yml` file. Open and edit the `stack.yml` provided with in the Python example code. Change `gateway` and `topic` as per your environment/needs.
+
+> **Note:** A key-value annotation under `topic` defines which VM event should trigger the function. A list of VM events from vCenter can be found [here](https://code.vmware.com/doc/preview?id=4206#/doc/vim.event.VmEvent.html). Multiple topics can be specified using a `","` delimiter syntax, e.g. "`topic: "VmPoweredOnEvent,VmPoweredOffEvent"`".
+
+```yaml
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP # replace with your vCenter Event Broker Appliance environment
+functions:
+  pdinvoke-fn:
+    lang: python3
+    handler: ./handler
+    image: vmware/veba-pagerduty-invoke:latest
+    environment:
+      write_debug: true
+      read_debug: true
```

Comment:
You might want to add the `insecure_ssl` option here as well so it's apparent to the user